src/Libraries/RevitNodes/Elements/Room.cs

```diff
@@ -21,6 +21,7 @@ public class Room : Element
         /// <summary>
         /// Internal reference to the Revit Element
         /// </summary>
+        [SupressImportIntoVM]
```

Comment:
If the method, property or class is internal then you don't need this attribute. But if it's public then you do need to specify this attribute when you are using Autodesk.Revit.DB namespace type in your method/property.